Makefile

```diff
@@ -1,3 +1,5 @@
+DHOST = $(shell echo $$(docker-machine ip))
```

Comment:
No. It will only exist if you are running Mac/Windows, apparently. I suspect we need to modify the above such that it falls back to 127.0.0.1 when the `docker-machine ip` command fails to work. The problem is that in my env (Mac), I can't run the tests locally without specify the above. I'll get it fixed.